src/installers/helm.ts

```diff
@@ -182,6 +182,7 @@ error: E_COMMAND_FAILED`)
   async prepareCheHelmChart(flags: any, cacheDir: string) {
     const srcDir = path.join(flags.templates, '/kubernetes/helm/che/')
     const destDir = path.join(cacheDir, '/templates/kubernetes/helm/che/')
+    await remove(destDir)
```

Comment:
That makes sense but shouldn't we do the same for every installer (operator and minishift addon)?

Reply:
Let me check. I faced this issue only with helm installer.

Reply:
You're totally right!
I thought that the error happened in `mkdirp` but actually an error happens when there are some conflicting files.
![Screenshot_20190823_145921](https://user-images.githubusercontent.com/5887312/64426679-e325ab00-d0b7-11e9-9c44-2acfbc829d25.png)

I'm adding `removeDir` for operator and minishift addon as well.

Reply:
Fixed in https://github.com/che-incubator/chectl/pull/301/commits/0b01a4166c490d3a1f6e8be23ea761f80ab1e9ec